documentation/book/ref-metrics-config-files.adoc

```diff
@@ -0,0 +1,39 @@
+
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='ref-metrics-config-files-{context}']
+
+= Example Metrics files
+
```

Comment:
I think you should explain where these examples can be found (in the installation and examples directory).

Reply:
You can find the example metrics configuration files in the installation ZIP file available for this release from the {ZipDownload}.